modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/repository/capabilities/PortalCapabilityLocatorImpl.java

```diff
@@ -204,9 +197,21 @@ public WorkflowCapability getWorkflowCapability(
 			DLFileVersionServiceAdapter.create(documentRepository));
 	}
 
-	@Override
-	public void invalidate() {
-		_clearLiferayDynamicCapabilities();
+	public class PortalCapabilityCacheRegistryItem
+		implements CacheRegistryItem {
+
+		@Override
+		public String getRegistryName() {
+			Class<?> clazz = getClass();
+
+			return clazz.getName();
+		}
```

Comment:
Hey @GabrielHenriquedev, in this case, you should hard code the `PortalCapabilityLocatorImpl` className as in the examples in the previous commits. Thx